backend/de.metas.business/src/main/sql/postgresql/system/20-de.metas.business/5622730_sys_gh12393_UpdateInvoiceProcessingServiceCompany.sql

```diff
@@ -0,0 +1,15 @@
+UPDATE c_doctype
+SET docsubtype = 'SI'
+FROM InvoiceProcessingServiceCompany serviceCompanySettings
+WHERE (1 = 1)
+  AND serviceCompanySettings.ServiceInvoice_DocType_ID = c_doctype.c_doctype_id
+  AND serviceCompanySettings.isactive = 'Y'
+  AND c_doctype.docbasetype != 'API';
+
+UPDATE InvoiceProcessingServiceCompany
+SET ServiceInvoice_DocType_ID = docType.c_doctype_id
+FROM c_doctype docType
+WHERE (1 = 1)
+  AND docType.docbasetype = 'API'
+  AND docType.docsubtype = 'SI'
+  AND docType.isactive = 'Y';
```

Comment:
```suggestion
```

i can't see why we would need this. AFAIU at this point there aren't any c_doctype with docsubtype = 'SI' that are not yet assigned to `InvoiceProcessingServiceCompany`

Reply:
The idea was to use the new doctype everywhere from now on...

but anyway, since we will drop the first script, I think we def need this, right?